backend_tsc/src/entitites/catalogue.ts

```diff
@@ -0,0 +1,29 @@
+import {Schema,model,Model} from "mongoose"
+import {IMongo_Entity} from "./mongo_entity";
+import Site, {ISite} from "./site";
+import {cascade_related} from "../utilities/trigger_factory";
+
+
+//Schema
+export interface ICatalogue extends IMongo_Entity{
+   name: string,
+   description: string
+}
```

Comment:
@Martyrshot @DanielleDPowell  - I'm wondering if we wouldn't be better off moving the entities (ICatalogue, ISite, IArtifact, etc.) to a single file or at least into files seperate from their MongoDB schema. Just want to manage separation of concerns. Thoughts?